src/api/app/models/event/build.rb

```diff
@@ -2,7 +2,7 @@ module Event
   class Build < Base
     self.description = 'Package has finished building'
     self.abstract_class = true
-    payload_keys :project, :package, :sender, :repository, :arch, :release, :readytime, :srcmd5,
+    payload_keys :project, :package, :sender, :repository, :arch, :release, :readytime, :srcmd5, :disturl
```

Comment:
You are missing a comma here

Reply:
Thanks for the review. I've force pushed a fixed patch.